### PR TITLE
Use container-fluid and set max-width to 1600px

### LINKED ISSUE #1177: Allow the main container to expand further
Our current designs for desktop were done with a width of 1608px, but we're currently using bootstrap's default `container` class for the main content, which is often substantially smaller than that on most screens. The result is that the UI looks a little cramped.

To allow more breathing room, especially on the show page, let's try changing the `.container` to a `.container-fluid` and then set a max-width of around 1600px so that it doesn't become too wide on very large screens.

### PR DESCRIPTION
**Pull request by marlo-longley**
Closes #1177

Looks like:
Screenshot 2024-08-15 at 13 16 25
Screenshot 2024-08-15 at 13 16 38

### CODE REVIEW

**Comment by marlo-longley** [comment]
@dbranchini I would love a visual check on this PR!

File: app/assets/stylesheets/earthworks.css [hunk 1]
```diff
@@ -2,6 +2,12 @@
   --stanford-50-black: #979694;
 }
 
+/* Selects both #main-content and navbar containers */
+ .container-fluid {
+  max-width:  1600px;
```
Comment (jcoyne) [inline app/assets/stylesheets/earthworks.css:7]: This seems like you're going to confuse people. A fluid container is defined as not having any max width and no padding.  Are you sure you don't want to add `.container-xxl` instead?

Comment (marlo-longley) [reply app/assets/stylesheets/earthworks.css:7]: Thanks for reviewing Justin. I was not part of any discussions about the ticket parameters. I just know that the design called for a breakpoint at 1600 and 60px of padding.

If adding this CSS to the `container-fluid` class feels like it's going against good practice, maybe we could add an additional class and style that.

Comment (jcoyne) [reply app/assets/stylesheets/earthworks.css:7]: Something like this: https://github.com/sul-dlss/earthworks/pull/1183

Comment (marlo-longley) [reply app/assets/stylesheets/earthworks.css:7]: Thanks. This looks good but potentially does not align with the designer spec -- I am not sure how strict that is. Let me check.

Comment (marlo-longley) [reply app/assets/stylesheets/earthworks.css:7]: Hi @jcoyne - appreciate that PR example. I just chatted with the workcycle team and learned that we actually do want the behavior of `container-fluid` at all sizes, so even at those smaller breakpoints we want to take up 100% minus 60px.

So I am going to try a different approach than those breakpoints and update this PR.

Comment (marlo-longley) [reply app/assets/stylesheets/earthworks.css:7]: OK, basically it's the same approach, just not targeting `.container-fluid` ...

File: app/assets/stylesheets/earthworks.css [hunk 2]
```diff
@@ -2,6 +2,12 @@
   --stanford-50-black: #979694;
 }
 
+/* Selects both #main-content and navbar containers */
+#main-container, .navbar-search > * {
+  max-width:  1600px;
+  padding: 0 60px;
```
Comment (jcoyne) [inline app/assets/stylesheets/earthworks.css:8]: could you set the variable instead?
```css
--bs-gutter-width-x: 120px
```

**Review by dnoneill — APPROVED** [review]

**Pull request merged**